client/src/templates/Challenges/classic/editor.tsx

```diff
@@ -446,6 +446,12 @@ const Editor = (props: EditorProps): JSX.Element => {
         newLine.run();
       }
     );
+    // @ts-ignore
```

Comment:
They're following the pattern we used in the previous `_standaloneKeybindingService` calls - at the top of this list is a comment which explains the linter and compile-check need to be disabled because we are accessing a private method.